cephci/prereq.py

```diff
@@ -218,23 +217,18 @@ def setup_private_container_registry(
     if not add_cert_to_trusted_list(installer):
         return False
 
-    # Validate the truster list is updated
-    if not validate_trusted_list(installer):
-        return False
-
     # Step 6: Copy the certificate to any nodes that will access the private registry for installation and update the
     # trusted list
     if not copy_cert_to_secondary_node(installer, nodes):
         return False
 
     # Step 7: Login to the registry
-    for node in nodes:
-        Registry(node).login(
-            registry=registry, username=reg_username, password=reg_password
-        )
+    Registry(installer).login(
```

Comment:
One doubt here, don't we need to enable the registry login from all the required nodes? Let's say the monitoring services are deployed on a different node, will this help in pulling or redeploying the mon services on that node? Please ignore if I got it wrong

Reply:
+1

Reply:
@pranavprakash20 we are configuring private registry on a single node. 

The following command will be executed to add any images to private registry: 
podman run -v /opt/registry/certs:/certs:Z -v /opt/registry/certs/domain.cert:/certs/domain.cert:Z --rm registry.redhat.io/rhel8/skopeo:8.5-8 skopeo copy  --remove-signatures --src-creds myusername:mypassword1 --dest-cert-dir=./certs/ --dest-creds myregistryusername:myregistrypassword1 docker://registry.redhat.io/rhceph/rhceph-6-rhel9:latest docker://admin.lab.redhat.com:5000/rhceph/rhceph-6-rhel9:latest

Given that, we'll only require to login to a specific registry from which we are going to add images to the private registry. 

Hence, login to registry (Not private registry here) will only need on a single node. 

Please refer to the doc for more info:

https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/6/html/installation_guide/red-hat-ceph-storage-installation#configuring-a-private-registry-for-a-disconnected-installation_install

Reply:
Noted. Thanks